metpy/xarray.py

```diff
@@ -370,6 +370,8 @@ def parse_cf(self, varname=None, coordinates=None):
             # If we found them, create a lat/lon projection as the crs coord
             if has_lat and has_lon:
                 var.coords['crs'] = CFProjection({'grid_mapping_name': 'latitude_longitude'})
+                log.warning('Found lat/lon values, assuming latitude_longitude '
```

Comment:
Should we mention that we haven't already found a projection, like "No projection found but found lat/lon values"?

Reply:
That is already given with the warning that is issued when it doesn't find any valid projection details in the file, just above this part in the code.

```python
try:
    proj_var = self._dataset.variables[proj_name]
    var = self._dataset.variables[proj_name]
except KeyError:
    log.warning('Could not find variable corresponding to the value of '
                'grid_mapping: {}'.format(proj_name))
```

Reply:
So both warnings would show up if there isn't a CRS but lat/lon values exist - which I think would be the ideal message to users here. @dopplershift do you want any changes to this still?

Reply:
Ah, nope, we're good.